### DISCUSSION TITLE
关于“数字主权”

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#85] [日常讨论]
What is digital sovereignty and why is Europe so interested in it?

- https://www.weforum.org/agenda/2021/03/europe-digital-sovereignty/

**As open source is a strategic enabler of digital sovereignty, process innovation and a source of attractiveness and goodwill.**

**Comment by will-ww** [comment]
还有“科技主权”，也是欧盟现在非常关心的话题：https://sciencebusiness.net/system/files/reports/2020-TECH-1.pdf

**Comment by wangleo61** [comment]
谢谢王老师，这个有意思，科技主权，跟咱们的自主可控是不是有点关联

Leo

________________________________
发件人: Will Wang ***@***.***>
发送时间: Wednesday, May 11, 2022 10:21:44 AM
收件人: kaiyuanshe/ONES ***@***.***>
抄送: Subscribed ***@***.***>
主题: Re: [kaiyuanshe/ONES] 关于“数字主权” (Discussion #85)


还有“科技主权”，也是欧盟现在非常关心的话题：https://sciencebusiness.net/system/files/reports/2020-TECH-1.pdf

―

**Reply by will-ww** [reply]
> 谢谢王老师，这个有意思，科技主权，跟咱们的自主可控是不是有点关联 Leo
> 

我觉得很有关联，欧盟现在这块做的很多，也很重视，也是希望自主独立的发展~

**Comment by wangleo61** [comment]
谢谢王老师解惑

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Will ***@***.***>
发送时间: Wednesday, May 11, 2022 6:49 PM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] 关于“数字主权” (Discussion #85)


谢谢王老师，这个有意思，科技主权，跟咱们的自主可控是不是有点关联 Leo

我觉得很有关联，欧盟现在这块做的很多，也很重视，也是希望自主独立的发展~

―
